app/components/Input/Interactions.tsx

```diff
@@ -1,6 +1,6 @@
 import type { FormEvent } from 'react';
 import { useRef } from 'react';
-import { BookmarkSvg, LikeSvg, StarSvg, ViewSvg } from './SVG';
+import { BookmarkSvg, LikeSvg, StarSvg, ViewSvg } from '@components/SvgComponents/InteractionsSVG';
```

Comment:
Isn't that more coplicated path than `'./SVG'`? You should add an alias to SvgComponents folder